### PR TITLE
K8SPG-61 reflect new cr structure in docs

### PR DESCRIPTION
**Pull request by fiowro**

### CODE REVIEW

File: source/operator.rst [hunk 1]
```diff
@@ -32,133 +105,206 @@
...
+| **Value**       | string                                                                                    |
++-----------------+-------------------------------------------------------------------------------------------+
+| **Example**     | ``perconalab/percona-postgresql-operator:main-ppg13-pgbackrest-repo``                     |
++-----------------+-------------------------------------------------------------------------------------------+
+| **Description** | The Docker image for the :ref:`BackRest repository<backups.pgbackrest.repository>`        |
++-----------------+-------------------------------------------------------------------------------------------+
+|                                                                                                             |
++-----------------+-------------------------------------------------------------------------------------------+
+|                 | .. _backup-resources-requests-memory:                                                     |
+|                 |                                                                                           |
```
Comment (spron-in) [inline source/operator.rst:306]: No CPU requests in the doc.

File: source/operator.rst [hunk 2]
```diff
@@ -171,546 +317,707 @@
...
+| **Key**         | `backup.storages.<storage-name>.type <operator.html#backup-storages-type>`_               |
 +-----------------+-------------------------------------------------------------------------------------------+
-| **Value**       | boolean                                                                                   |
+| **Value**       | string                                                                                    |
 +-----------------+-------------------------------------------------------------------------------------------+
-| **Example**     | ``false``                                                                                 |
+| **Example**     | ``s3``                                                                                    |
 +-----------------+-------------------------------------------------------------------------------------------+
-| **Description** | Enables or disables TLS verification for pgBackRest                                       |
+| **Description** | The cloud storage type used for backups. Only ``s3`` type is supported                    |
```
Comment (spron-in) [inline source/operator.rst:425]: Where is GCS that we added in this release?

File: source/operator.rst [hunk 3]
```diff
@@ -171,546 +317,707 @@
...
 +-----------------+-------------------------------------------------------------------------------------------+
 |                                                                                                             |
 +-----------------+-------------------------------------------------------------------------------------------+
-|                 | .. _spec-shutdown:                                                                        |
+|                 | .. _backup-storages-s3-endpointurl:                                                       |
 |                 |                                                                                           |
-| **Key**         | `shutdown <operator.html#spec-shutdown>`_                                                 |
+| **Key**         | `backup.storages.s3.<storage-name>.endpointURL                                            |
+|                 | <operator.html#backup-storages-s3-endpointurl>`_                                          |
```
Comment (spron-in) [inline source/operator.rst:432]: The key is `backup.storages.<storage_name>` , not `backup.storages.s3.<storage-name>`

same issue below in the doc.

File: source/operator.rst [hunk 4]
```diff
@@ -32,133 +105,206 @@
...
-| **Key**         | `backrestRepoPath <operator.html#spec-backrestrepopath>`_                                 |
+| **Key**         | `walStorage.volumeSpec.matchLabels <operator.html#walstorage-volumespec-matchlabels>`_    |
 +-----------------+-------------------------------------------------------------------------------------------+
 | **Value**       | string                                                                                    |
 +-----------------+-------------------------------------------------------------------------------------------+
 | **Example**     | ``""``                                                                                    |
 +-----------------+-------------------------------------------------------------------------------------------+
-| **Description** | Optional reference to the location of the pgBackRest repository                           |
+| **Description** | `Labels are key-value pairs attached to objects                                           |
+|                 | <https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/>`_             |
```
[the remark this answers is not in the document]
Comment (spron-in) [reply source/operator.rst:263]: I see this issue multiple times. Please fix everywhere.

File: source/operator.rst [hunk 5]
```diff
@@ -171,546 +317,707 @@
...
 +-----------------+-------------------------------------------------------------------------------------------+
-| **Value**       | string                                                                                    |
+| **Value**       | label                                                                                     |
 +-----------------+-------------------------------------------------------------------------------------------+
-| **Example**     | ``""``                                                                                    |
+| **Example**     | ``pg-cluster-label: cluster1``                                                            |
 +-----------------+-------------------------------------------------------------------------------------------+
-| **Description** | Supplemental groups for the PostgreSQL write-ahead log storage                            |
+| **Description** | `Labels are key-value pairs attached to objects                                           |
+|                 | <https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/>`_             |
```
Comment (spron-in) [inline source/operator.rst:903]: Again the same template, but the description would be different here. This one actually sets the labels for pgReplica Pods.

File: source/operator.rst [hunk 6]
```diff
@@ -171,546 +317,707 @@
...
 |                 |                                                                                           |
-| **Key**         | `pmm.serverHost <operator.html#pmm-serverhost>`_                                          |
+| **Key**         | `pgReplicas.<replica-name>.expose.loadBalancerSourceRanges                                |
+|                 | <operator.html#pgreplicas-expose-loadbalancersourceranges>`_                              |
 +-----------------+-------------------------------------------------------------------------------------------+
-| **Value**       |  string                                                                                   |
+| **Value**       | string                                                                                    |
 +-----------------+-------------------------------------------------------------------------------------------+
-| **Example**     |  ``monitoring-service``                                                                   |
+| **Example**     | ``""``                                                                                    |
```
Comment (spron-in) [inline source/operator.rst:943]:
```suggestion
| **Example**     | ``"10.0.0.0/8"``                                                                                    |
```

File: source/operator.rst [hunk 7]
```diff
@@ -171,546 +317,707 @@
...
 +-----------------+-------------------------------------------------------------------------------------------+
-|                 | .. _pmm-serveruser:                                                                       |
+|                 | .. _pgreplicas-expose-annotations:                                                        |
 |                 |                                                                                           |
-| **Key**         | `pmm.serverUser <operator.html#pmm-serveruser>`_                                          |
+| **Key**         | `pgReplicas.<replica-name>.expose.annotations                                             |
+|                 | <operator.html#pgreplicas-expose-annotations>`_                                           |
 +-----------------+-------------------------------------------------------------------------------------------+
-| **Value**       | string                                                                                    |
+| **Value**       | label                                                                                     |
```
Comment (spron-in) [inline source/operator.rst:955]: is label a specific value type? isn't it a string?

File: source/operator.rst [hunk 8]
```diff
@@ -171,546 +317,707 @@
...
 +-----------------+-------------------------------------------------------------------------------------------+
-| **Example**     | ``cluster1-pmm-secret``                                                                   |
+| **Example**     | ``pg-cluster-label: cluster1``                                                            |
 +-----------------+-------------------------------------------------------------------------------------------+
-| **Description** | Name of the `Kubernetes Secret object                                                     |
-|                 | <https://kubernetes.io/docs/concepts/configuration/secret/#using-imagepullsecrets>`_ for  |
-|                 | the PMM Server password                                                                   |
+| **Description** | `Labels are key-value pairs attached to objects                                           |
+|                 | <https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/>`_             |
```
Comment (spron-in) [inline source/operator.rst:974]: this sets the label for the service.

File: source/operator.rst [hunk 9]
```diff
@@ -32,133 +105,206 @@
...
+`Backup Section <operator.html#operator-backup-section>`_
+--------------------------------------------------------------------------------
+
+The ``backup`` section in the
+`deploy/cr.yaml <https://github.com/percona/percona-xtradb-cluster-operator/blob/main/deploy/cr.yaml>`__
+file contains the following configuration options for the regular
+Percona Distribution for PostgreSQL backups.
+
+.. tabularcolumns:: |p{2cm}|p{13.6cm}|
+
```
[the remark this answers is not in the document]
Comment (fiowro) [reply source/operator.rst:279]: afaik cr.yaml doesn't include it...

**Review by spron-in — APPROVED** [review]

**Pull request merged**